.sequelizerc

```diff
@@ -0,0 +1,8 @@
+var path = require('path');
```

Comment:
For consistency throughout your codebase, use ES6 keyword `const` instead of `var`